### PR TITLE
fix: eslint

### PR DESCRIPTION
**Pull request by septs**
closes #

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "lint-staged": {
     "*": "prettier --write --ignore-unknown",
     "packages/**/*.{ts,tsx,js,jsx}": "eslint -c packages/.eslintrc.json --fix",
-    "packages/maskbook/**/*": "gulp locale-kit --sync-keys --remove-unused-keys"
+    "packages/maskbook/**/*": "sh -c 'gulp locale-kit --sync-keys --remove-unused-keys'"
```
Comment (septs) [inline package.json:27]: macOS and Linux workaround

Windows: no idea

Comment (septs) [reply package.json:27]: CC @Jack-Works

Comment (Jack-Works) [reply package.json:27]: Why do you need a `sh`?

Comment (septs) [reply package.json:27]: you need gulp allow input any parameter

see https://github.com/okonet/lint-staged#packagejson-example

Comment (Jack-Works) [reply package.json:27]: Maybe you can write a small `.js` file to execute that command without extra input received

File: packages/maskbook/src/plugins/Wallet/SNSAdaptor/GasNowDialog.tsx [hunk 2]
```diff
@@ -163,7 +163,7 @@ export function GasNowDialog() {
                             {gasNow || i === 2 ? (
                                 <>
                                     <div>
-                                        {i == 2 ? (
+                                        {i === 2 ? (
```
Comment (guanbinrui) [inline packages/maskbook/src/plugins/Wallet/SNSAdaptor/GasNowDialog.tsx:166]: cc @zhouhanseng

**Review by guanbinrui — DISMISSED** [review]

**Review by Jack-Works — DISMISSED** [review]

File: packages/scripts/src/bin/precommit-hook-localekit.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const { spawnSync } = require('child_process')
+const { resolve } = require('path')
+
+spawnSync('npx', ['gulp', 'locale-kit', '--sync-keys', '--remove-unused-keys'], {
```
Comment (septs) [inline packages/scripts/src/bin/precommit-hook-localekit.js:4]: need handle exit code issue

**Review by Jack-Works — DISMISSED** [review]

File: packages/scripts/src/bin/precommit-hook-localekit.js [hunk 4]
```diff
@@ -1,8 +1,16 @@
...
     shell: true,
 })
+if (error) {
+    console.error(error)
+}
+if (status === null || status === 0) {
+    process.exit(0)
+} else {
+    process.exit(status)
+}
```
Comment (septs) [inline packages/scripts/src/bin/precommit-hook-localekit.js:16]:
```suggestion
process.exit(status === null || status === 0 ? 0 : status)
```

**Review by Jack-Works — APPROVED** [review]

**Review by zhouhanseng — APPROVED** [review]

**Pull request merged**